pkg/okteto/constants/constants.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2022 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package constants
+
+const (
```

Comment:
Can we move them to `pkg/okteto/const.go`. It seems odd to me to have one file with constants and a package with even more constant in the same package

Reply:
We need this constants to be in a different package than `okteto`, and with `golang` we need to create a folder for it.

We could move the new `constants` package to another place, or we could try to move the 3 constants in `pkg/okteto/const.go` into this one

Reply:
if we move constants out the okteto pkg would this grow as config pkg ? maybe we can have the not exported constants within the file that is being used and the exported ones at the global pkg?

Reply:
@teresaromero the idea of this package is to solve some unnecessary imports we were having. The idea is to not use it as much as we can (added a quick description about this in the `doc.go`). We should never import something from inside this package, since it wouldn't be constant and could lead us to a problematic package like the one you describe (`config`).

I like the idea "have the not exported constants within the file that is being used". I tried to do that as much as I could. If you find one that could be moved, please notify me. Also, we could let that to a follow up pr since this already adds a lot of value by its own